lib/versioning/swift/index.ts

```diff
@@ -29,21 +29,36 @@ const {
 
 export const isValid = (input: string): boolean =>
   !!valid(input) || !!validRange(toSemverRange(input));
+
 export const isVersion = (input: string): boolean => !!valid(input);
-const getSatisfyingVersion = (versions: string[], range: string): string =>
-  maxSatisfying(
-    versions.map((v) => v.replace(regEx(/^v/), '')), // TODO #12071
-    toSemverRange(range)
-  );
-const minSatisfyingVersion = (versions: string[], range: string): string =>
-  minSatisfying(
-    versions.map((v) => v.replace(regEx(/^v/), '')), // TODO #12071 #12070
-    toSemverRange(range)
-  );
-const isLessThanRange = (version: string, range: string): boolean =>
-  ltr(version, toSemverRange(range));
-const matches = (version: string, range: string): boolean =>
-  satisfies(version, toSemverRange(range));
+
+function getSatisfyingVersion(
+  versions: string[],
+  range: string
+): string | null {
+  const normalizedVersions = versions.map((v) => v.replace(regEx(/^v/), ''));
+  const semverRange = toSemverRange(range);
+  return semverRange && maxSatisfying(normalizedVersions, semverRange);
```

Comment:
I think this now returns `false` instead of `null` ?

Reply:
`semverRange` is either string or null, though made it clear with code anyway